homework/list.js

```diff
@@ -6,17 +6,24 @@
   var _wrapper = document.querySelector('#note-list-wrapper');
 
   function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
+    var li = new list();
```

Comment:
And another issue is you **shouldn't create a instance in a file for definition**. You should leave the work in a file to bootstrap the whole program.